jakartaee10/JakartaEE10ReleasePlan.md

```diff
@@ -4,3 +4,167 @@
+Many other component Specifications are introducing updates which are binary compatible and, thus, will only require a Minor version update.
+Regardless of the scope of changes, all artifacts for the component and Platform Specifications will be affected -- Specifications, APIs, TCKs, and Compatible Implementations.
+
+### Core Profile ([issue](https://github.com/eclipse-ee4j/jakartaee-platform/issues/387))
+Complementing the Web Profile, a new profile called the Core Profile is being proposed for Jakarta EE 10.
+The goal of the Core Profile is to define a subset of the Jakarta EE technologies which target microservices development and runtimes.
+It is still not clear if the creation of the Core Profile will be ready for Jakarta EE 10.
+A separate release plan will be developed, circulated, and balloted for the Core Profile.
+
+## Java SE 11 ([issue](https://github.com/eclipse-ee4j/jakartaee-platform/issues/331))
+Java SE 11 will become the minimum runtime supported by Jakarta EE compatible implementations.
+
+### API Source and Target Level
+If a component Specification is planning a Major or Minor version update for Jakarta EE 10, then the recommendation would be to recompile and distribute the specification’s APIs at the Java SE 11 source and target levels.
+This is not a requirement since there may be reasons to continue compiling and distributing the APIs at an earlier Java source or target level.
+
+**Note:** The Platform API uber jar files will all be compiled at the Java SE 11 source/target levels.
+This is consistent with past practices.
+These uber jar files will be deprecated for Jakarta EE 10 to allow for possible removal in a future release.
+
+### TCK Source Level
+The TCKs will be compiled at the Java SE 11 level.
+In order to allow for runtimes beyond Java SE 11, the TCK will also need to be (successfully) executed with Java SE 17.
+
+#### Signature Tests
+In Jakarta EE 9.1, the framework for performing API Signature tests was updated to make it easier to test Signatures across multiple Java levels.
+The framework and/or API tests may need to be tweaked as we continue to learn and experiment with future levels of Java.
+
+### Compatible Implementation (CI) Source Level
+How a Compatible Implementation supports the Java SE 11 runtime (or above) will be left as a vendor-defined solution.
+
+## JPMS Module Info classes ([issue](https://github.com/eclipse-ee4j/jakartaee-platform/issues/329))
+Suggestions on support for JPMS modules were introduced in the Jakarta EE 9 Platform Specification.
+It is desired to firm up these suggestions into requirements.
+Several discussions via Issues, Mailing Lists, and the Platform call have resulted in these requirements for Jakarta EE 10.
+ 
+* Each individual component specification API provides at least one `module-info.class` following a specified set of conventions:
+ * spec module name is `jakarta.<specName>`
```

Comment:
I would really like to get the official short name of the specification in common use. I think we have some inconsistencies here.

Reply:
We have this [names.adoc](https://github.com/jakartaee/specification-committee/blob/master/names.adoc).  I could reference that.  And, if it's not accurate, then we could get it updated.  But, this is where I look for the proper names.